src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/ResolveSpecificCommentAuthorizationHandler.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers
+{
+    public class ResolveSpecificCommentRequirement : IAuthorizationRequirement
+    {
+    }
+
+    public class
```

Comment:
Maybe amend the formatting here from:

```
public class
        ResolveSpecificCommentAuthorizationHandler : AuthorizationHandler<ResolveSpecificCommentRequirement, Comment>
```

to

```
public class ResolveSpecificCommentAuthorizationHandler 
    : AuthorizationHandler<ResolveSpecificCommentRequirement, Comment>
```

Is that any better?  Meh...